src/install/redis/host/atlas.mdx

```diff
@@ -0,0 +1,13 @@
+---
+componentType: default
+headingText: Prepare Atlas 
+---
+
+
```

Comment:
We should split this up into two separate ideas, and consider front loading the action first rather than front loading the tool you're using to do the action. for example:

> Create a new user using the Atlas CLI. You'll assign `clusterMonitor` and `readanyDatabase` roles to this user. Be sure to adjust the `username` and `password` as needed: